.github/workflows/build-and-test.yml

```diff
@@ -1169,6 +1169,18 @@ jobs:
       - uses: actions/download-artifact@v2
         with:
           name: stt-tflite-3.7-aarch64.whl
+      - uses: actions/download-artifact@v2
+        with:
```

Comment:
Python38?

Reply:
I’ve selected the wrong line hahah but it’s the same as above, we extract artefacts that don’t exist since we remove python 3.8.

Reply:
Fixed